docs/components/dartpad/tabs/fixed/main.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      home: DefaultTabController(
+        length: 3,
+        child: Scaffold(
+          appBar: AppBar(
+            title: Text('Fixed Tabs'),
+            automaticallyImplyLeading: false,
+            backgroundColor: Color(0xff5808e5),
+            bottom: TabBar(
```

Comment:
Make `indicatorColor` white to better match material baseline

Reply:
Done